jwst/background/background_step.py

```diff
@@ -58,13 +58,38 @@ def process(self, input, bkg_list):
                 result = background_sub.subtract_wfss_bkg(
                                 input_model, bkg_name, wlrange_name)
                 result.meta.cal_step.back_sub = 'COMPLETE'
-
             else:
+                # check if input data is NRS_IFU
+                tolerance = 1.0e-15
+                result = input_model.copy()
+                do_sub = True
+                if input_model.meta.exposure.type in ["NRS_IFU"]:
+                    # check if GWA_XTILT & GWA_YTILT values of source
+                    # background are the same. If not skip step
+                    input_xtilt = input_model.meta.instrument.gwa_xtilt
+                    input_ytilt = input_model.meta.instrument.gwa_ytilt
+                    for bkg_file in bkg_list:
+                        bkg_model = datamodels.ImageModel(bkg_file)
+                        bkg_xtilt = bkg_model.meta.instrument.gwa_xtilt
+                        bkg_ytilt = bkg_model.meta.instrument.gwa_ytilt
+                        xdiff = np.absolute(bkg_xtilt - input_xtilt)
```

Comment:
The tilts are scalars, so you can use Python's `abs` function, and you wouldn't need to import `numpy`.